omod/src/main/webapp/resources/scripts/field/fingerprintM2sys.js

```diff
@@ -1,6 +1,8 @@
-function m2sysEnroll() {
+function m2sysEnroll(button) {
+    toggleFingerprintButtonDisplay(button);
     jq.getJSON('/' + OPENMRS_CONTEXT_PATH + '/registrationapp/field/fingerprintM2sys/enroll.action')
     .success(function(data) {
+        toggleFingerprintButtonDisplay(button);
```

Comment:
What about the failed messages. I think we should also toggle the button when  we get .error() or .fail(). Perhaps you can use .always() or .done()

Reply:
You're right, I'll add the .error() case too.